DynamicModeler/Logic/vtkSlicerDynamicModelerExtrudeTool.cxx

```diff
@@ -104,32 +108,45 @@ vtkSlicerDynamicModelerExtrudeTool::vtkSlicerDynamicModelerExtrudeTool()
   /////////
   // Parameters
 
-  ParameterInfo parameterExtrusionScale(
-    "Extrusion scale",
-    "Length of normals, line or point distance is scaled by this value to get the offset surface position. Use negative value for reversing extrusion direction.",
-    "ExtrusionScale",
+  ParameterInfo parameterLengthMode(
+    "Length mode",
+    "Can be fixed length or scaled length. Scaled mode is ignored for planes",
```

Comment:
Why planes are always absolute?

Reply:
What should be the 'Relative' behavior? I did not know so, I did not implement it
Maybe, it could be came as MarkupPoint but using the plane origin

Reply:
Please see my comment at the bottom of the pull request, I added illustrative pictures